server/testutil/testregistry/testregistry.go

```diff
@@ -126,7 +126,7 @@ func (r *Registry) PushRandomImage(t *testing.T) (string, gcr.Image) {
 	}
 	image, err := crane.Image(files)
 	require.NoError(t, err)
-	return r.Push(t, image, "test"), image
+	return r.Push(t, image, "test:latest"), image
```

Comment:
Curious why this change is needed?

Reply:
Not strictly necessary. I just wanted to make sure we used a name with a tag in the tests I wrote. Could move that bit to the specific tests I'm worried about.